tox.ini

```diff
@@ -0,0 +1,29 @@
+[tox]
+envlist = py{27,35}
```

Comment:
This is a template for reusable Django apps, and my first use of it is for an app that we want to use in edx-platform.  The output has to support 2.7 or we won't be able to use it in our largest service until we eventually get around to the Python 3 update for it.